Improve robustness of salt-master and upgrade (#3125)
scality/metalk8s: CHANGELOG.md

# CHANGELOG

## Release 2.7.2 (in development)
### Enhancements

> - Improve salt-master and upgrade robustness on environment that are a bit slow

Reviewer: Since it's customer-facing:

```suggestion
- Improve Salt master and cluster upgrade stability in slow environments
```

Author: Done
